src/dcat-ap/index.ts

```diff
@@ -26,7 +26,7 @@ export function getDataStreamDcat201(options: IDcatAPOptions) {
 
```

Comment:
I think the portalUrl should be built-up before you get here (i.e., passed into the function).

Reply:
Done. I also renamed it to `orgBaseUrl` to make it more clear.